draft/spec/index.html

```diff
@@ -620,10 +620,10 @@ <h2>Basic Structure</h2>
                 </dd>
                 <dt><code>digestAlgorithm</code></dt>
                 <dd>
-                    The digest algorithm used for calculating digests within the OCFL Object. This
-                    <span id="E039">MUST</span> be either <code>sha512</code> or <code>sha256</code>, and
-                    <span id="W006">SHOULD</span> be <code>sha512</code>. See thead
-                    <a href="#digest-algorithms">section on Digests</a> for more information.
+                    The digest algorithm used for calculating digests for content-addressing within the OCFL Object
+                    and for the <a href="#inventory-digest">Inventory Digest</a>. This <span id="E039">MUST</span> be
+                    the algorithm used in the <code>manifest</code> and <code>state</code> blocks, see the
```

Comment:
The digest algorithm is used for:
- content addressing,
- inventory digests, *and*
- the fixity block

Did you intentionally omit the fixity block usage?

Reply:
Yes, because the fixity uses an independently defined and likely different set of digest algorithms. The fixity block use is not content addressing